android/src/main/java/com/epicshaggy/biometric/NativeBiometric.java

```diff
@@ -283,9 +339,55 @@ private String decryptString(String stringToDecrypt, String KEY_ALIAS) throws Ge
     }
 
 
+    private KeyPair generateKeyPair() throws GeneralSecurityException {
+        //Create the start and expiry date for the key
+        // TODO: confirm the expiration date
+        GregorianCalendar startDate = new GregorianCalendar();
+        GregorianCalendar endDate = new GregorianCalendar();
+        endDate.add(Calendar.YEAR, 1);
```

Comment:
For now, I think we don't need to add `endDate` since the expirytime will be handled in BE